includes/payments/class-edd-payment.php

```diff
@@ -962,12 +962,19 @@ public function add_download( $download_id = 0, $args = array(), $options = arra
 		$discount   = $args['discount'];
 		$subtotal   = $amount;
 		$tax        = $args['tax'];
+		$fees       = 0;
+
+		if ( ! empty( $args['fees'] ) ) {
+			foreach ( $args['fees'] as $fee ) {
+				$fees += $fee['amount'] < 0 ? $fee['amount'] : 0;
+			}
+		}
 
 		if ( edd_prices_include_tax() ) {
 			$subtotal -= round( $tax, edd_currency_decimal_filter() );
 		}
 
-		$total      = $subtotal - $discount + $tax;
+		$total      = $subtotal - $discount + $tax + $fees;
```

Comment:
In `edd_get_cart_content_details` we add the fees to `$subtotal` before it is included in `$total`. For consistency's sake and future-proofing, can we do it the same please?